indico/migrations/versions/20170905_1521_640584a3987e_add_event_groups_to_acls.py

```diff
@@ -0,0 +1,200 @@
+"""Add event groups to acls
+
+Revision ID: 640584a3987e
+Revises: f1eee7b4880a
+Create Date: 2017-09-05 15:21:13.323181
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '640584a3987e'
+down_revision = 'f1eee7b4880a'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('attachment_principals', sa.Column('event_group_id', sa.Integer()), schema='attachments')
+    op.add_column('folder_principals', sa.Column('event_group_id', sa.Integer()), schema='attachments')
+    op.add_column('contribution_principals', sa.Column('event_group_id', sa.Integer()), schema='events')
+    op.add_column('principals', sa.Column('event_group_id', sa.Integer()), schema='events')
+    op.add_column('session_principals', sa.Column('event_group_id', sa.Integer()), schema='events')
+    op.add_column('settings_principals', sa.Column('event_group_id', sa.Integer()), schema='events')
+    op.create_index(None, 'attachment_principals', ['event_group_id'], schema='attachments')
+    op.create_index(None, 'folder_principals', ['event_group_id'], schema='attachments')
+    op.create_index(None, 'contribution_principals', ['event_group_id'], schema='events')
+    op.create_index(None, 'principals', ['event_group_id'], schema='events')
+    op.create_index(None, 'session_principals', ['event_group_id'], schema='events')
+    op.create_index(None, 'settings_principals', ['event_group_id'], schema='events')
+    op.create_foreign_key(None, 'attachment_principals', 'groups',
+                          ['event_group_id'], ['id'],
+                          source_schema='attachments', referent_schema='events')
+    op.create_foreign_key(None, 'folder_principals', 'groups',
+                          ['event_group_id'], ['id'],
+                          source_schema='attachments', referent_schema='events')
+    op.create_foreign_key(None, 'contribution_principals', 'groups',
+                          ['event_group_id'], ['id'],
+                          source_schema='events', referent_schema='events')
+    op.create_foreign_key(None, 'principals', 'groups',
+                          ['event_group_id'], ['id'],
+                          source_schema='events', referent_schema='events')
+    op.create_foreign_key(None, 'session_principals', 'groups',
+                          ['event_group_id'], ['id'],
+                          source_schema='events', referent_schema='events')
+    op.create_foreign_key(None, 'settings_principals', 'groups',
+                          ['event_group_id'], ['id'],
+                          source_schema='events', referent_schema='events')
+    op.execute('''
+        ALTER TABLE attachments.attachment_principals DROP CONSTRAINT ck_attachment_principals_valid_enum_type;
```

Comment:
Using raw SQL here is much less ugly than writing alembic/sqlalchemy statements for it, especially since the CHECK constraint updates are not autogenerated (so like this the dbdiff output can be copied almost as-is, only thing I changed was removing the `SET search_path` stuff in favor of explicit schemas in table names)